src/webpack/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
Can probably add some nice bits and bobs to this and perhaps the Tapestry logo - although... serving the project's favicon is quite nice now I think about it.